src/GovUk.Education.ExploreEducationStatistics.Admin/Services/UserManagementService.cs

```diff
@@ -239,5 +317,37 @@ private void SendInviteEmail(string email)
 
             _emailService.SendEmail(email, template, emailValues);
         }
+
+        private void SendNewReleaseRoleEmail(Guid userId, IdTitlePair publication, IdTitlePair release, EnumExtensions.EnumValue role)
+        {
+            var uri = _configuration.GetValue<string>("AdminUri");
+            var template = _configuration.GetValue<string>("NotifyReleaseRoleTemplateId");
+            var email = _usersAndRolesDbContext.Users.FirstOrDefault(x => x.Id == userId.ToString())?.Email;
+
+            var emailValues = new Dictionary<string, dynamic>
+            {
+                {"url", "https://" + uri + "/publication/" + publication.Id + "/release/" + release.Id + "/summary"},
```

Comment:
`$"https://{uri}/publication/{publication.Id}/release/{release.Id}/summary"` might be nicer to read?

Reply:
Ah, i thought i had done that already